core/arch/arm/plat-rockchip/platform_config.h

```diff
@@ -44,10 +44,17 @@
 #error "Unknown platform flavor"
 #endif
 
-#define CONSOLE_UART_BASE	UART2_BASE
-#define CONSOLE_UART_SIZE	UART2_SIZE
-#define CONSOLE_BAUDRATE	1500000
-#define CONSOLE_UART_CLK_IN_HZ	24000000
+/*
+ * Early Console UART definitions
+ * OP-Tee will switch to a devicetree-based console later during
+ * startup, based on the actual board-specific console settings.
+ * So the fixed early console should only be enabled for debugging
+ * purposes during development:
+#define EARLY_CONSOLE_UART_BASE		UART2_BASE
+#define EARLY_CONSOLE_UART_SIZE		UART2_SIZE
+#define EARLY_CONSOLE_UART_CLK_IN_HZ	24000000
+#define EARLY_CONSOLE_BAUDRATE		1500000
+ */
```

Comment:
Perhaps you could avoid changing the behavior for the existing platform by conditionally defining those?
```C
#if defined(PLATFORM_FLAVOR_rk322x)
#define ...
#endif
```